client/app/scripts/superdesk/editor/editor.js

```diff
@@ -21,6 +21,19 @@ function EditorService() {
     this.addEventListeners = addEventListeners;
     this.removeEventListeners = removeEventListeners;
 
+    // Undo Redo related variables
+    this.historyBodyHtml = [];
```

Comment:
This can be refactored to a dict where the key is the field name, this will allow to support undo/redo automatically on all the places where the editor is used.
The map would look like `{'headline': {'stack': [], 'index': 1}}`